bcgov/workbc-main
# [WR-738] Import labour market and others

- Update IA sheet as per latest changes
- Update latest clean database
- Generate Labour Market Monthly node in IA
- Adjust path alias configuration for Labour Market Monthly
- Ensure migration runs correctly
